assets/styl/_layouts/list.twig

```diff
@@ -0,0 +1,44 @@
+<p><b>Example 1</b> - Simple column list:</p>
+<div class="list co_list-demo">
```

Comment:
You should mention somewhere `co_list-demo` and `co_list-demo-item` classes are here only for style demo.
And are not part of colette

Reply:
done 👍